crates/llm-ls/Cargo.toml

```diff
@@ -22,10 +22,10 @@ tree-sitter-bash = "0.20"
 tree-sitter-c = "0.20"
 tree-sitter-cpp = "0.20"
 tree-sitter-c-sharp = "0.20"
-tree-sitter-elixir = "0.1"
```

Comment:
What happened here? 😄

Reply:
As mentioned in the PR description, this version of the module used C++, so I changed it to use the git repository, where the module is rewritten in C, so that it can compile with musl.

Reply:
This is why the current musl version does not work -- it requires `libstdc++.so.6`.

Reply:
Probably should leave a comment above each of these tree-sitter dependencies to state why they are pulling in a specific version from github.